src/ert/simulator/batch_simulator.py

```diff
@@ -103,9 +103,7 @@ def callback(*args, **kwargs):
             )
 
         for key in results:
-            ens_config.addNode(
-                GenDataConfig(name=key, input_file=f"{key}_%d", report_steps=[0])
```

Comment:
Does this make it optional? Because everyone is doing this at the moment, so we need to be backwards compatible.

Reply:
As far as I can see yep. I think if someone were to give a suffix'ed control, that suffix would just be the filename? Not sure if they can/will explicitly pass on report steps to BatchSimulator from Everest?

Entry point is 
```
everest/simulator/simulator.py:__init__()
results_def = self._get_results_def(ever_config)
```
and it does not directly access/use `report step`.

Reply:
Good point, no, the report step concept is not really in everest, just still small piece of it.

Reply:
Come to think of it, they often have a `name` that is without `_0`, then they need to produce a file with `_0`, let me double check that we dont break something here. Maybe we have to do some validation in everest.

Reply:
Made a follow up issue here: https://github.com/equinor/everest-models/issues/52